src/Database/Schema/TableSchemaInterface.php

```diff
@@ -51,6 +51,13 @@ interface TableSchemaInterface extends SchemaInterface
      */
     public const TYPE_DATETIME = 'datetime';
 
+    /**
+     * Datetime with fractional seconds column type
+     *
+     * @var string
+     */
+    public const TYPE_DATETIME_FRACTIONAL = 'datetimef';
```

Comment:
A single extra character is not easy to spot. If `datetimefractional` seems to long then we can shorten it to `datetimefract`.

Reply:
I went back and forth on this.  I decided to submit the PR with the smallest version and get feedback.

Reply:
Normally it's best to maintain consistency with other constants and also keep the constant's value similar to it's name, so I would suggest keeping the value `datetime_fractional`.

Reply:
Sure, that is my preference as well.